tests/infra_test.go

```diff
@@ -289,11 +289,9 @@ var _ = Describe("Infrastructure", func() {
 			By("Checking the collected metrics")
 			keys := getKeysFromMetrics(metrics)
 			for _, key := range keys {
-				if strings.HasPrefix(key, "kubevirt_vm_metrics_") {
```

Comment:
Why do we not need the if statement anymore?

Reply:
It is now guaranteed (or at least that's the intention) by newRandomVMIWithMetrics, see below. We now take metrics with the user-supplied prefix, which is a stronger guarantee because is going to be more specific than the "kubevirt" prefix we used previously.